src/async/importer/CsvImporter.js

```diff
@@ -0,0 +1,35 @@
+import path from 'path';
+import parse from 'csv-parse'
+import parseCsvSync from 'csv-parse/lib/sync';
+import {
+    promisify
+} from 'util';
+
+import Importer from './Importer'
+
+const parseCsv = promisify(parse);
+
+
+export default class CsvImporter extends Importer {
+
+    async readDir(dirPath) {
+        const files = await super.readDir(dirPath);
+        const csvFiles = await files.filter(file => path.extname(file) === '.csv');
+        return csvFiles.reduce(async(prevValue, value) => {
```

Comment:
Here you are reducing an array into a new array with initial value `[]`. You can make such action easier with `Array#map`.

You can also chain 2 last lines into
```
files
  .filter()
  .map()
```
You can use single reduce instead of filter + map to optimize iterations count, but usually its not that important